apps/www/src/lib/plate/demo/plugins/removeOnDeleteForwardPlugin.ts

```diff
@@ -0,0 +1,14 @@
+import { PlatePlugin } from '@udecode/plate-common';
+import { ELEMENT_HR } from '@udecode/plate-horizontal-rule';
+import { ELEMENT_IMAGE } from '@udecode/plate-media';
+import { RemoveOnDeleteForwardPlugin } from '@udecode/plate-select';
+
+export const removeOnDeleteForwardPlugin: Partial<
```

Comment:
Can be removed

Reply:
Remove as per https://github.com/udecode/plate/pull/2687#discussion_r1349769808